code-style/javascript/README.md

```diff
@@ -26,23 +26,23 @@ Frameworks / Libraries
 
 JavaScript has a rich ecosystem of libraries. Here are some of the tools we like to use.
 
-- [React][react] - A view library for building user interfaces
-- [Vue][vue] - Another view library for building user interfaces
-- [Redux][redux] - A state management library
-- [Axios][axios] - An [isomorphic][isomorphic] HTTP client
-- [Lodash][lodash] - A utility library packed full of useful functionality
-- [Moment][moment] - A date/time library
-- [Chartist][chartist] - A responsive chart library
+- [React][react] – A view library for building user interfaces
+- [Vue][vue] – Another view library for building user interfaces
```

Comment:
With the React and Vue descriptions both being essentially the same, would it be helpful to add some information about when we use one vs the other?

Reply:
I'd like to seek input from some of the devs that are more familiar with Vue and do a follow up PR for this.